src/sbd-md.c

```diff
@@ -1279,11 +1279,10 @@ int servant_md(const char *diskname, int mode, const void* argp)
 		}
 	}
  out:
-	free(s_header);
 	free(s_node);
 	free(s_mbox);
+	free(s_header);
```

Comment:
Shouldn't s_mbox be first?

Reply:
Allocation is done header, mbox and then node - so doing it the opposite direction here ...

Reply:
... at least that far the theory why it seemed to be more pleased with what I changed around line 560. It actually never complained about this here and iirc at some point it even stopped complaining about the stuff around 560. Had the impression it was some kind of tuning when they brought gcc into fedora. Just to avoid it next time ;-)